package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember test"
+    "test": "ember exam --split=3 --weighted --parallel"
```

Comment:
Do we want to override this in `circle.yml` instead? Maybe change to `ember exam` here and then that config won't cause problems locally for people.

What do you think @rileytaylor?